src/buildCommon.js

```diff
@@ -367,6 +367,16 @@ var makeVList = function(children, positionType, positionData, options) {
     return vlist;
 };
 
+// Converts verb group into body string, dealing with \verb* form
+var makeVerb = function(group, options) {
+    var text = group.value.body;
+    if (group.value.star) {
+        text = text.replace(/ /g, '\u2423');  // Open box
```

Comment:
Is that symbol included in the tt font? Come to think of it, do we want to complain if the input contains any symbol not present in our tt font, as we can't guarantee consistent rendering or correct metrics for these?